pkgs/applications/networking/insync/v3.nix

```
@@ -0,0 +1,77 @@
+{ stdenv
+, lib
+, fetchurl
+, makeWrapper
+, dpkg
+, glibc
+, glib
+, libxcb
+, libGL
+, nss
+, libthai
+, wayland
+, alsaLib
+, qt5
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "insync";
+  version = "3.2.4.40856";
+  src =
+    if stdenv.hostPlatform.system == "x86_64-linux" then
+      fetchurl {
+        url = "http://s.insynchq.com/builds/${name}_${version}-focal_amd64.deb";
+        sha256 = "1bvqbbrfn5784nmb2qaflm1rzczqhvghhb6y5zaxrapyhygxbcis";
+      }
+    else throw "${name} is not supported on ${stdenv.hostPlatform.system}";
```

```suggestion
  src = fetchurl {
        url = "http://s.insynchq.com/builds/${name}_${version}-focal_amd64.deb";
        sha256 = "1bvqbbrfn5784nmb2qaflm1rzczqhvghhb6y5zaxrapyhygxbcis";
      };
```

platform is only set to this so it is not expected to work anymwhere else.

---

oh yeah good point. Fixing.